gov2/cloudfront/CreateDistribution/CreateDistribution_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// snippet-start:[gov2.cloudfront.CreateDistribution]
+
+package main
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/cloudfront"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+)
+
+func TestCreateDistribution(t *testing.T) {
+	thisTime := time.Now()
+	nowString := thisTime.Format("2023-02-05 15:04:05 Sunday")
+	t.Log("Starting unit test at " + nowString)
```

Comment:
This is an integration test (because it relies on AWS as an external resource), not a unit test.